apps/test/unit/lib/util/urlHelpersTest.js

```diff
@@ -23,6 +23,27 @@ describe('pegasus()', () => {
   });
 });
 
+describe('studio()', () => {
+  describe('from pegasus', () => {
+    stubWindowPegasus({
+      STUDIO_URL: '//test-studio.code.org'
+    });
+
+    it('gives an absolute studio url', () => {
+      expect(studio('/relative-path')).to.equal('//test-studio.code.org/relative-path');
+    });
+  });
+
+  describe('from studio', () => {
+    stubWindowPegasus(undefined);
+
+    it('returns the relative URL if not', () => {
```

Comment:
nit: remove `if not`. I think it's more clear without:

`studio from studio returns the relative URL` 
vs. 
`studio from studio returns the relative URL if not`